up_tamer/solver.py

```diff
@@ -355,11 +355,12 @@ def _convert_plan(self, tproblem: pytamer.tamer_problem, plan: 'up.plan.Plan') -
             pytamer.tamer_ttplan_add_step(ttplan, step)
         return ttplan
 
-    def validate(self, problem: 'up.model.Problem', plan: 'up.plan.Plan') -> bool:
+    def validate(self, problem: 'up.model.Problem', plan: 'up.plan.Plan') -> 'up.solvers.results.ValidationResult':
         assert self.supports(problem.kind)
         tproblem = self._convert_problem(problem)
         tplan = self._convert_plan(tproblem, plan)
-        return pytamer.tamer_ttplan_validate(tproblem, tplan) == 1
+        value = pytamer.tamer_ttplan_validate(tproblem, tplan) == 1
+        return ValidationResult(ValidationResultStatus.VALID if value else ValidationResultStatus.INVALID)
```

Comment:
Minor comment :smile: 

```suggestion
        is_valid = pytamer.tamer_ttplan_validate(tproblem, tplan) == 1
        return ValidationResult(ValidationResultStatus.VALID if is_valid else ValidationResultStatus.INVALID)
```